compiler/modules/front/front.bal

```diff
@@ -176,6 +178,14 @@ function importPartPrefixes(ScannedModule scanned, (ModuleExports|string?)[] res
             resolved = ioLibFunctions;
             partial = true;
         }
+        else if moduleId == BALLERINA_LANG_INT {
```

Comment:
This doesn't seem quite right. This is processing import declarations, but we don't have import declarations for these prefixes.  Why do we need this?

Reply:
My bad, will remove these.